bin/run.sh

```diff
@@ -22,5 +22,7 @@ cp -R $input_folder/* .
 
 find . -mindepth 1 -type f | grep 'Test.java' | xargs -I file sed -i "s/@Ignore(.*)//g;s/@Ignore//g;" file
 
-java -jar /opt/test-runner/autotest-runner.jar $problem_slug
+echo -e "test {\n  maxHeapSize = '2G'\n  reports.html.required = false\n}" >> build.gradle
+timeout 20 gradle test --offline --no-daemon --warning-mode=none --no-watch-fs --console=plain 2> gradle-test.err
```

Comment:
I moved the external process execution from Java to the command line for a couple of reasons:

 - faster iteration cycles (no need to recompile to change options passed to gradle)
 - easier to change
 - log handling for debug purposes
 
 I think that I prefer this here, but I can revert this change if required, not fussy about it.

Reply:
I think those are all good arguments in favor of running this in bash.